src/memory_targets/memory_target_numa.c

```diff
@@ -41,22 +41,6 @@ static umf_result_t numa_initialize(void *params, void **memTarget) {
 
 static void numa_finalize(void *memTarget) { free(memTarget); }
 
-static const umf_os_memory_provider_params_t
-    UMF_OS_MEMORY_PROVIDER_PARAMS_DEFAULT = {
-        // Visibility & protection
-        .protection = UMF_PROTECTION_READ | UMF_PROTECTION_WRITE,
-        .visibility = UMF_VISIBILITY_PRIVATE,
-
-        // NUMA config
-        .nodemask = NULL,
-        .maxnode = 0, // TODO: numa_max_node/GetNumaHighestNodeNumber
```

Comment:
you could "save" these TODOs  (perhaps move them or file an issue for them), unless it's already been resolved 😉

Reply:
Those TODO are not longer relevant actually. This entire initialization code was just copied from somewhere with those TODOs but in this file, we specifically set maxnode to number of memory targets.

Reply:
And in tests, we already have a mechanism for finding maxnode.